vignettes/covidHubUtils-overview.Rmd

```diff
@@ -67,12 +67,12 @@ We have identified two central use cases for downloading data using `load_foreca
 
 2. Downloading all available forecasts for selected models that are submitted on a set of dates. This is achieved by using a vector of dates in `dates` parameter.
 
-Below are some examples of reading in data. We start by loading the `covidHubUtils` package and the `tidyverse`.
+Below are some examples of reading in data. We start by loading the `covidHubUtils` package.
 
 ```{r message=FALSE}
 library(covidHubUtils)
 library(doParallel)
-library(tidyverse)
+library(ggplot2)
```

Comment:
Another place where we can be more specific about the required library to avoid a big tidyverse install.